eng/helixpublish.proj

```diff
@@ -0,0 +1,122 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>runtime</DotNetCliPackageType>
+    <!-- Set to match global.json.tools.dotnet by pre-build.ps1 -->
+    <DotNetCliVersion>$(MicrosoftNetCoreAppVersion)</DotNetCliVersion>
+    <DotNetCliRuntime>win-x86</DotNetCliRuntime>
+    <DotNetCliRuntime Condition="'$(Platform)' == 'x64'">win-x64</DotNetCliRuntime>
+
+    <EnableXUnitReporter>true</EnableXUnitReporter>
+    <EnableAzurePipelinesReporter>true</EnableAzurePipelinesReporter>
+    <FailOnMissionControlTestFailure>true</FailOnMissionControlTestFailure>
+
+    <!-- Semi-colon delimited list of commands that are executed before every test.-->
+    <HelixPreCommands>$(HelixPreCommands);</HelixPreCommands>
+
+    <!-- These are the list of Windows versions that we run tests on-->
+    <HelixTargetQueues>Windows.10.Amd64.Open;Windows.7.Amd64.Open;Windows.10.Amd64.ServerRS5.Open</HelixTargetQueues>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <HelixCorrelationPayload Include="$(ArtifactsDir)\test\$(Configuration)\$(Platform)" /> 
+    <HelixWorkItem Include="Xaml">
+      <Timeout>00:20:00</Timeout>
+      <Command>call %HELIX_CORRELATION_PAYLOAD%\runtests.cmd /Area=Xaml</Command>
+    </HelixWorkItem>
+   
+   <!--
```

Comment:
You have the ItemGroup that defines the DRT set.  Can you use that ItemGroup here to build your work items by batching?

Reply:
yeah that's a good suggestion. i was thinking about it, but I really want to codegen that DRT set based off DrtList.xml, so I figured it could wait until then. although there's no real reason for waiting.